Source/WebCore/page/scrolling/mac/ScrollingTreeScrollingNodeDelegateMac.mm

```diff
@@ -66,6 +66,22 @@
     ThreadedScrollingTreeScrollingNodeDelegate::updateFromStateNode(scrollingStateNode);
 }
 
+void ScrollingTreeScrollingNodeDelegateMac::updateScrollbarsFromStateNode(const ScrollingStateScrollingNode& scrollingStateNode, CALayer** horizontalLayer, CALayer** verticalLayer)
```

Comment:
This function doesn't update scrollbars, it just fetches layers, so could use a more appropriate name.

WebKit style is `CALayer **` (note spacing for ObjC types)

Reply:
Fixed.